pkg/registry/apps/apps.go

```diff
@@ -55,7 +58,8 @@ func ProvideRegistryServiceSink(
 		logger.Debug("Investigations backend is enabled")
 		providers = append(providers, investigationAppProvider)
 	}
-	if features.IsEnabledGlobally(featuremgmt.FlagGrafanaAdvisor) {
+	if features.IsEnabledGlobally(featuremgmt.FlagGrafanaAdvisor) &&
+		!strings.Contains(strings.Join(grafanaCfg.DisablePlugins, ","), "grafana-advisor-app") {
```

Comment:
Why not just `!slices.Contains(grafanaCfg.DisablePlugins, ""grafana-advisor-app")`?

Reply:
the signature is `slices.Contains(string, string)`, `DisabledPlugins` is a `string[]` so I need to join it first

Reply:
aah, sorry, right, we can use that